src/components/App.css

```diff
@@ -0,0 +1,5 @@
+.App {
+  display: flex;
+  flex-direction: column;
+  width: 700px;
+}
```

Comment:
You can center the app, for a symmetry with the page 
``` 
  margin: auto 
```

Reply:
Hey Shey,

Thanks for the feedback. I will center the project on the next milestone submission👍